### ISSUE TITLE
scctl: resetting renderer by friendly name might fail if sender has the same name

### ISSUE THREAD

**Issue by joerg-krause** [medoc92/upmpdcli#110]
`scctl -x <friendly name>` does not necessarily stops the Receiver mode if the Sender device has the same name as the Renderer device, e.g. "UpMpd". This happens in my case when Songcast Sender and Receiver are running on the same hardware device.

This is not really an issue with scctl, but with libupnpp as the device discovery using a friendly name only returns one device. If this device is the Sender instead of the Renderer (which is bearing the Receiver service) `scctl -x`.

I know, that using the UDN is the safer approach. However, I wondered if the device discovery using a friendly name should find all devices having the same name. So, more like an issue about libupnpp, but as `scctl` is part of upmpdcli I decided to post the issue here.

Feel free to close the issue by saying: "Please use UDN instead of friendly name when using scctl".

**Comment by medoc92** [comment]
Even if I added an easy interface to return  all the devices with the same name , what are you going to do next ? You still don't know which to stop !

So either the user makes the effort to have unique friendly names (which is useful, really: you want to know where the renderer is located !), and they can use the friendly names, or they don't and only the UDNs can be used.

It would be  trivial to write a method to return all the devices with the same friendly name, using the current discovery interface: so much so that I'm not sure that it belongs in the library itself.

**Comment by joerg-krause** [comment]
It's not really about having different renderers using the same friendly name in a network, but about having the renderer and the sender service running on the same hardware. In my case, I used the hostname when starting upmpdcli and mpd2sc as the friendly name, e.g:

![screenshot from 2017-06-23 08-59-44](https://user-images.githubusercontent.com/6870896/27470038-581889f0-57f2-11e7-8992-211dab2e463c.png)

In this case `scctl -x UpMpd` only works if the renderer device was found before the sender device.  Therefore, I changed the friendly name for the sender device:

![screenshot from 2017-06-23 08-58-46](https://user-images.githubusercontent.com/6870896/27470043-5a26a042-57f2-11e7-978b-547336c3ce60.png)

I noticed that you take care about that issue by appending "UxSender" to the friendly name in the scmakempdsender script. I did not in my custom script.
